musicbot/bot.py

```diff
@@ -1689,7 +1689,7 @@ async def _cmd_play(
     ):
         player = _player if _player else None
 
-        if permissions.summonplay and not player:
+        if permissions.summonplay:
```

Comment:
If we don't have `and not player` the constant summon messages when using the play command occur.

Reply:
Ah ok, not sure how it disapeared but I'll add it back in.